Game/ui/left panel/ResourceDisplay/item_display_row.gd

```diff
@@ -75,6 +79,7 @@ func update_storage_cap(new_cap: float) -> void:
 
 ## Internal function to render the text
 func update_view() -> void:
+	storage_cap = Model.get_storage_cap(multiplayer.get_unique_id(), item_type)
```

Comment:
Why was this change made? I figured it doesn't update often so we can update it when it actually changes via `update_storage_cap()`, and this undoes that work.